server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
I'd like to maintain the possibility to report another metric under the flattr name. However, I need your knowledge to give an appropriate name. At first, I believed that the figure represented a number of donators, but [this](https://api.flattr.com/rest/v2/things/391332) leads me to believe that there is a separate field for that, `flattrs_user_count`. What does a flattr represent? How can a user flattr a product multiple times? My research makes me think that they are monthly contributions off a fixed budget, which would make `contributions` a suitable name. It seems odd, since it is absolutely not proportional to the amount of money received, nor to the number of consumers, which makes it seem like an odd metric to show. I'd like to know more.